Lib/test/test_asyncio/test_ssl.py

```diff
@@ -1,5 +1,4 @@
 import asyncio
-import asyncio.sslproto
```

Comment:
Could this one possibly be here intentionally for some side effect? @kumaraditya303

Reply:
Still waiting for @kumaraditya303 to wake up. :-)

Reply:
No it can removed. Only `asyncio.events` has side effect of registering fork handler and that too is already imported by `asyncio` so no worries.